### DISCUSSION TITLE
How can I get back to the new design?

### DISCUSSION THREAD

**Discussion by Desoroxxx** [darkreader/darkreader#13424] [Q&A] [answered]
I accidentally switched to the old design and can't find a way to switch back

![image](https://github.com/user-attachments/assets/3cb852ac-b4db-4dfd-bbcd-af2d6194b20d)
![image](https://github.com/user-attachments/assets/8bcb4c19-0884-4967-821b-23f96fa1ccaf)
![image](https://github.com/user-attachments/assets/0f577344-42b3-4e61-ae40-9e13773fdfde)

**Comment by Desoroxxx** [comment]
Anyone?

**Comment by Myshor** [comment, marked as answer]
![obraz](https://github.com/user-attachments/assets/8da92a5c-5be6-4f9c-86ed-0a9675132926)
